libnavigation-copilot/src/main/java/com/mapbox/navigation/copilot/MapboxCopilot.kt

```diff
@@ -0,0 +1,66 @@
+package com.mapbox.navigation.copilot
+
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+import com.mapbox.navigation.copilot.internal.PushStatusObserver
+import com.mapbox.navigation.core.MapboxNavigation
+import com.mapbox.navigation.core.lifecycle.MapboxNavigationApp
+import com.mapbox.navigation.core.lifecycle.MapboxNavigationObserver
+import java.util.concurrent.CopyOnWriteArraySet
+
+/**
+ * MapboxCopilot.
+ */
+object MapboxCopilot : MapboxNavigationObserver {
```

Comment:
```suggestion
@ExperimentalPreviewMapboxNavigationAPI
object MapboxCopilot : MapboxNavigationObserver {
```

Was there a reason to call the individual functions experimental rather than the top level?

Reply:
I think it is better to make the whole class experimental, so that in the future we will be able to move it to a separate package, rename or completely remove if necessary. With the current implementation we will be able to change only its functions.

Reply:
Great point. Let me quickly fix and re-push.